commands/zinter.md

```diff
@@ -0,0 +1,21 @@
+The difference with `ZINTERSTORE` is that the result is returned rather than 
+stored.
+
+For a description of the `WEIGHTS` and `AGGREGATE` options, see `ZUNIONSTORE`.
+
+@return
```

Comment:
shouldn't this have something similar to ZRANGE?
```
@array-reply: list of elements in the specified range (optionally with
their scores, in case the `WITHSCORES` option is given).
```

Reply:
ohh, my bad. it has it.